### PR TITLE
Add quentin

### PR DESCRIPTION
**Pull request by Sporarum**

### CODE REVIEW

File: _data/team-members.yml [hunk 1]
```diff
@@ -210,3 +210,11 @@ members:
     photo: /resources/img/jedrzej.jpg
     contact_for: [scastie]
     active: true
+  - member: quentin
+    name: Quentin Bernet
+    title: Software Development - Intern
+    email: quentin.bernet@epfl.ch
+    github: Sporarum
+    photo: /resources/img/quentin.jpeg
+    contact_for: [Dotty, Documentation Improvements]
```
Comment (julienrf) [inline _data/team-members.yml:219]:
```suggestion
    contact_for: [dotty, documentationImprovements]
```

These values have to match the labels defined here: https://github.com/scala/scala.epfl.ch/tree/main/_projects

**Review by julienrf — COMMENTED** [review]
I am not 100% whether this is mandatory or not, but the other team members have all uploaded two sizes of profile pictures, could you please do the same?

**Comment by Sporarum** [comment]
> I am not 100% whether this is mandatory or not, but the other team members have all uploaded two sizes of profile pictures, could you please do the same?

I reuploaded the pictures, they are now in .png with resolution 150x150 and 300x300 to conform with other pictures

**Review by julienrf — APPROVED** [review]
Thank you Quentin!

**Comment by bishabosha** [comment]
not sure what's happening, Quentin appears in the team page, but not the index page

**Comment by julienrf** [comment]
I see an issue with the picture, which is `png` in the repo, but `jpeg` in the data. Could you please use jpeg everywhere, Quentin?

**Comment by Sporarum** [comment]
Shouldn't I use png everywhere ?
As this is the format used by all the other prictures

**Comment by bishabosha** [comment]
you can update the team_members yml to point to the png file

**Comment by julienrf** [comment]
I don’t really understand why the others use png, but jpeg is the most appropriate format to use here.

**Pull request merged**